### PR TITLE
fix for issue #82, unknown host exception reading filehttps://stackov…

### PR DESCRIPTION
**Pull request by david-gibbs-ig**
Please see [Issue#82](https://github.com/esanchezros/quickfixj-spring-boot-starter/issues/82)

### CODE REVIEW

File: quickfixj-spring-boot-starter/src/test/java/io/allune/quickfixj/spring/boot/starter/connection/SessionSettingsLocatorTest.java [hunk 1]
```diff
@@ -43,9 +43,8 @@ public void shouldLoadDefaultFromSystemProperty() throws URISyntaxException {
 		SessionSettings settings = sessionSettingsLocator.loadSettings("classpath:quickfixj.cfg", null, null, null);
 		assertThat(settings).isNotNull();
 
-		URL resource = SessionSettingsLocatorTest.class.getResource("/quickfixj.cfg");
-		File file = Paths.get(resource.toURI()).toFile();
-		settings = sessionSettingsLocator.loadSettings(null, null, "file://" + file.getAbsolutePath(), null);
+		File file = new File(SessionSettingsLocatorTest.class.getClass().getResource("/quickfixj.cfg").getFile());
```
Comment (esanchezros) [inline quickfixj-spring-boot-starter/src/test/java/io/allune/quickfixj/spring/boot/starter/connection/SessionSettingsLocatorTest.java:46]: I think `getClass()` is redundant, and probably why the build is failing locally for me (Linux)

File: quickfixj-spring-boot-starter/src/test/java/io/allune/quickfixj/spring/boot/starter/connection/SessionSettingsLocatorTest.java [hunk 2]
```diff
@@ -43,9 +43,8 @@ public void shouldLoadDefaultFromSystemProperty() throws URISyntaxException {
 		SessionSettings settings = sessionSettingsLocator.loadSettings("classpath:quickfixj.cfg", null, null, null);
```
Comment (esanchezros) [inline quickfixj-spring-boot-starter/src/test/java/io/allune/quickfixj/spring/boot/starter/connection/SessionSettingsLocatorTest.java:43]: `URISyntaxException` can be removed

File: quickfixj-spring-boot-starter/src/test/java/io/allune/quickfixj/spring/boot/starter/connection/SessionSettingsLocatorTest.java (hunk 1 above)
Comment (david-gibbs-ig) [reply quickfixj-spring-boot-starter/src/test/java/io/allune/quickfixj/spring/boot/starter/connection/SessionSettingsLocatorTest.java:39]: Yes, it's redundant, removing.

**Comment by esanchezros** [comment]
Thanks David

**Pull request merged**